lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -360,21 +362,18 @@ public function getCreatePrimaryKeySQL(Index $index, $table)
     protected function getCreateColumnCommentSQL($tableName, $columnName, $comment)
     {
         if (strpos($tableName, '.') !== false) {
-            [$schemaSQL, $tableSQL] = explode('.', $tableName);
-            $schemaSQL              = $this->quoteStringLiteral($schemaSQL);
-            $tableSQL               = $this->quoteStringLiteral($tableSQL);
+            [$schemaName, $tableName] = explode('.', $tableName, 2);
```

Comment:
This implies that a schema name cannot have a dot.

Reply:
This is how it was, slight improvement when there are more dots, the remaining parts are not silently discarded.